libusb/core.c

```diff
@@ -902,6 +903,24 @@ void API_EXPORTED libusb_free_device_list(libusb_device **list,
 	free(list);
 }
 
+/** \ingroup libusb_dev
```

Comment:
oh, and the documentation misses `since version` clause
check for instance `libusb_get_port_numbers`

Reply:
Sure. I used the next upcoming version and build number (v1.0.27, `LIBUSB_API_VERSION >= 0x0100010B`), correct?

Also, removed the  `LIBUSB_HAS_GET_SESSION_DATA` preprocessor constant (users should check `LIBUSB_API_VERSION` for conditional compilation, just like with the rest of libusb API).